projects/demo-maps/src/app/route-components/route-example-custom-layers/route-map4.component.ts

```diff
@@ -275,7 +275,12 @@ export class RouteMap4Component implements OnInit, AfterViewInit {
       id: 'vectorTile',
       name: 'VectorTileLayer',
       visible: false,
-      popup: true,
+      popup: {
+        event: 'move',
```

Comment:
If the cursor is moved towards to the border of the map i get a event where i can detect if there are any features or layers for which the popup should be there.

So for this to work i have to calculate the position and the size of the map div in the Browser window and then check on move if the pixel coordinate is inside the map.
This is complicated because the header and the Aside may not always have the same size.

I you know any openlayers event which tells me that the cursor is moved outside the map, please let me know.

Reply:
There is now - finally!!!! - a brower event for this: 
https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event